### PR TITLE
Feat Model -- Added new facets to Dockerfile

### PR DESCRIPTION
**Pull request by nss10**
* Added new commands to the Dockerfile so as to run the python script

### CODE REVIEW

**Review by wangfan860 — APPROVED** [review]

File: generative_bayes_model/gbm-run-with-slack.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+        echo "Posting failed message to slack.."
+        payload="{\"attachments\": [{\"fallback\": \"JOB FAILED: gbm-run.sh job on ${gen3Env}\",\"color\": \"#ff0000\",\"title\": \"JOB FAILED: gbm-run.sh job on ${gen3Env}\",\"text\": \"Pod name: ${HOSTNAME}\",\"ts\": "$(date +%s)"}]}"
+        echo "${payload}"
+        curl -X POST --data-urlencode "payload=${payload}" "${slackWebHook}"
+    fi
+else
+    echo "JOB SUCCESS: gbm-run.sh"
+    if [[ -n "$slackWebHook" && "$slackWebHook" != "None" ]]; then
+        echo "Posting success message to slack.."
+        payload="{\"attachments\": [{\"fallback\": \"JOB SUCCESS: gbm-run.sh job on ${gen3Env}\",\"color\": \"#2EB67D\",\"title\": \"JOB SUCCESS: gbm-run.sh job on ${gen3Env}\",\"text\": \"Pod name: ${HOSTNAME}\",\"ts\": \"$(date +%s)\"}]}"
```
Comment (paulineribeyre) [inline generative_bayes_model/gbm-run-with-slack.sh:13]:
```suggestion
    echo "JOB FAILED: generative Bayes model"
    if [[ -n "$slackWebHook" && "$slackWebHook" != "None" ]]; then
        echo "Posting failed message to slack.."
        payload="{\"attachments\": [{\"fallback\": \"JOB FAILED: generative Bayes model job on ${gen3Env}\",\"color\": \"#ff0000\",\"title\": \"JOB FAILED: generative Bayes model job on ${gen3Env}\",\"text\": \"Pod name: ${HOSTNAME}\",\"ts\": "$(date +%s)"}]}"
        echo "${payload}"
        curl -X POST --data-urlencode "payload=${payload}" "${slackWebHook}"
    fi
else
    echo "JOB SUCCESS: generative Bayes model"
    if [[ -n "$slackWebHook" && "$slackWebHook" != "None" ]]; then
        echo "Posting success message to slack.."
        payload="{\"attachments\": [{\"fallback\": \"JOB SUCCESS: generative Bayes model job on ${gen3Env}\",\"color\": \"#2EB67D\",\"title\": \"JOB SUCCESS: generative Bayes model job on ${gen3Env}\",\"text\": \"Pod name: ${HOSTNAME}\",\"ts\": \"$(date +%s)\"}]}"
```

File: generative_bayes_model/gbm-run.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    echo "No S3 bucket provided (use env var S3_BUCKET)"
+    exit 1
+fi
+
+echo "Running pymc3_generative_model..."
+python3 pymc3_generative_model.py;
+
+echo "Copying results to S3 bucket..."
+if [[ -n "$S3_BUCKET" ]]; then
+    aws s3 sync "." "$S3_BUCKET/generative_bayes_model/" "--exclude" "*" "--include" "*.svg"
```
Comment (paulineribeyre) [inline generative_bayes_model/gbm-run.sh:13]: do you need all those `"`?

Comment (paulineribeyre) [reply generative_bayes_model/gbm-run.sh:13]: what's `--exclude *` for?

Comment (paulineribeyre) [reply generative_bayes_model/gbm-run.sh:13]: i think `aws s3 cp *.svg $S3_BUCKET/generative_bayes_model/` might work 🤔

Comment (nss10) [reply generative_bayes_model/gbm-run.sh:13]: I tried the same (but with quotes) and it didn't work! I thought quotes were a part of the syntax. So I did some reading and was told that, you will choose the src and destination paths, then you'll exclude everything using a wildcard and include only those files that you want to copy/sync. I can try a version of it without the quotes as well! Do you want me to?

Comment (paulineribeyre) [reply generative_bayes_model/gbm-run.sh:13]: i think we can use `cp` instead of `sync` for this use case

Comment (nss10) [reply generative_bayes_model/gbm-run.sh:13]: Pasting aws cli docs for reference -- https://docs.aws.amazon.com/cli/latest/reference/s3/#use-of-exclude-and-include-filters

**Review by paulineribeyre — APPROVED** [review]

**Pull request merged**